packages/web/src/components/createCell.tsx

```diff
@@ -122,10 +122,20 @@ export type CellSuccessProps<
  */
 export type DataObject = { [key: string]: unknown }
 
+type BeforeQueryHook<
+  CellProps extends Record<string, unknown>,
+  CellVariables extends Record<string, unknown>
+> = (props: CellProps) => {
+  variables: CellVariables
+  fetchPolicy: 'cache-and-network' | 'no-cache'
+  notifyOnNetworkStatusChange: boolean
+}
```

Comment:
beforeQuery's return will be passed to apollo's useQuery hook, so it can return more than these three props. We should probably try to extract the return type from apollo:

https://github.com/redwoodjs/redwood/blob/18e131e62b9749083ef4a6509004a078283139f4/packages/web/src/components/createCell.tsx#L272-L283